### ISSUE TITLE
Joint/Slider DontDrive defaults True in "AvatarCreator"

### ISSUE THREAD

**Issue by kazu0617** [Neos-Metaverse/NeosPublic#3153]
### Is your tweak request related to a problem? Please describe.

This checkbox avoids host-dependency issues related to the drive release mechanism.
And I think these issue also related: #1270 #2185 #2524

### Describe what would you like tweaked

in Slider/Joint component updates by host normally, but this will some issue in headless.
and this component also found in avatar creator, so some newcomers confusing for this issue.

so i think if this checkbox turns on will solve this problem.

Originally #2524 I was talking about turning it on in general, but then I wanted to know the use cases other than AvatarCreator, and in fact the discussion has stopped since then, so I cut out just AvatarCreator and made it an Issue.

### Describe alternatives you've considered

"DontDrive" turns on as Slider/Joint component uses, but little hard for newcomers.
Especially for avatar creators, it is very tedious to open the inspector and go through the trouble of working on it, and if this seems impossible, I would at least like to see a "Turn On Don't Drive (For headless)" checkbox.

**Comment by epicEaston197** [comment]
I don't know if I should make this into a separate issue but the Joint/Slider does not work in desktop mode when working with the Avatar Creator it's really difficult and I find myself deleting the slider component and replacing it with a grabbable component because rotating it precisely is almost impossible with a slider

**Comment by BlueCyro** [comment]
Necroposting on this issue because I feel it would be a perfectly fine change to simply tick "DontDrive" on the avatar creator/fullbody calibrator to true. I'm aware that the other two issues would implicitly fix the problem, but this would fix it more immediately. It wouldn't go against any of neos' philosophies since it's a behavior neos already has and supports. It would have negligible impact and would probably make a lot of people very happy.

As it stands there's a mod I made to fix the issue with the full body calibrator: https://github.com/RileyGuy/DontDriveMeMad
And one Sox made to fix the issue with the avatar creator: https://github.com/Sox-NeosVR/CompliantAvatarCreator

I feel like we really don't need to wait on the underlying issues if there's already a valid solution that won't impact development down the line.
